src/engraving/libmscore/stemslash.h

```diff
@@ -46,19 +46,23 @@ class StemSlash final : public EngravingItem
 
     Chord* chord() const { return (Chord*)explicitParent(); }
 
-    const mu::LineF& line() const { return m_line; }
-    void setLine(const mu::LineF& l) { m_line = l; }
+    struct LayoutData : public EngravingItem::LayoutData {
+        mu::LineF line;
+        double stemWidth = 0.0;
```

Comment:
I wonder to what extent the name `stemWidth` makes sense. It looks like it should actually be `stemSlashWidth`. But not sure if we should change that now.